examples/gcp_defaults/credentials.json

```diff
@@ -0,0 +1,4 @@
+{
+  "copy_and_paste": "your credentials file here",
+  "WARNING": "DO NOT FORGET TO ADD IT TO YOUR .gitignore FILE!"
+}
```

Comment:
I'd remove this file. The name implies something it is not

Reply:
done